t/shuffle_test.cc

```diff
@@ -115,15 +116,15 @@ INSTANTIATE_TEST_SUITE_P(BigWindows, WindowTest, Values(50, 99, 100, 1000));
 //  how we index the song list when picking randomly. It's hard to test
 //  that something is random :/.
 TEST(ShuffleChainTest, IsRandom) {
-    srand(4);
+    std::mt19937 rnd_engine{4};
```

Comment:
Use parens here to match the style of the rest of the file.

Reply:
Will do.